packages/next/client/image.tsx

```diff
@@ -584,6 +613,22 @@ export default function Image({
     objectFit,
     objectPosition,
   }
+
+  if (process.env.NODE_ENV !== 'production' && layout !== 'raw' && style) {
```

Comment:
Can we move this warning earlier so that it can live in the existing `if (process.env.NODE_ENV !== 'production')` with all the other warnings?

Reply:
done